setup.py

```diff
@@ -38,10 +34,6 @@ def modules():
 def packages():
     package_names = setuptools.find_packages()
 
-    # remove packages already contained in gardener-cicd-base
-    package_names.remove('ci')
-    package_names.remove('model')
-    package_names.remove('oci')
```

Comment:
I think the `oci` package is/was not actually part of `gardener-cicd-base` but instead it is part of `gardener-oci` (which is still declared as requirement for this package). So, I either suggest to still remove the `oci` package here or, if `gardener-oci` is not required anymore, to also remove it from the build.

Reply:
you are of course correct - good catch!